src/lib/utils/mem_ops.cpp

```diff
@@ -42,8 +50,11 @@ void deallocate_memory(void* p, size_t elems, size_t elem_size)
    if(mlock_allocator::instance().deallocate(p, elems, elem_size))
       return;
 #endif
-
+#if defined(BOTAN_TARGET_OS_HAS_ALLOC_CONCEAL)
+   ::freezero(p, elems * elem_size);
```

Comment:
The OpenBSD docs say "The malloc_conceal() and calloc_conceal() functions behave the same as malloc() and calloc() respectively, with the exception that the allocation returned is marked with the MAP_CONCEAL mmap(2) flag and **calling free() on the allocation will discard the contents explicitly.**" so AIUI we don't need this? Just calling `free` will cause the value to be zeroed as a side effect.

(Not that it matters in any case given we start the function off by zeroing the bits before calling any kind of free function)

Reply:
yes I missed the scrub_memory call...